source/faq.txt

```diff
@@ -4,12 +4,176 @@ FAQ
 
 .. default-domain:: mongodb
 
-Frequently Asked Questions
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-Lorem ipsum.
+General
+-------
+
+Why are there two versions of the classes? e.g. ``Mongo`` and ``MongoClient``?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``MongoDatabase``, ``MongoClient``, ``MongoCollection`` are the
+newer generation of classes and have better defaults for production use.
+
+For more information, see our blog post on `MongoClient
+<https://www.mongodb.com/blog/post/introducing-mongoclient>`_. 
+
+What's the difference between ``mongodb-driver-sync``, ``mongodb-driver`` or ``mongo-java-driver``?
```

Comment:
As of 4.0, we no longer publish `mongodb-driver` or `mongo-java-driver` artifacts.  I don't think they need to be mentioned at all, unless it's in an upgrade-to-4.0 guide.